lib/matplotlib/mlab.py

```diff
@@ -3136,17 +3136,17 @@ def get_type(item, atype=int):
     def get_justify(colname, column, precision):
         ntype = column.dtype
 
-        if np.issubdtype(ntype, str) or np.issubdtype(ntype, bytes):
+        if np.issubdtype(ntype, np.character):
             fixed_width = int(ntype.str[2:])
             length = max(len(colname), fixed_width)
             return 0, length+padding, "%s"  # left justify
 
-        if np.issubdtype(ntype, np.int):
+        if np.issubdtype(ntype, np.integer):
             length = max(len(colname),
                          np.max(list(map(len, list(map(str, column))))))
             return 1, length+padding, "%d"  # right justify
 
-        if np.issubdtype(ntype, np.float):
```

Comment:
This worked correctly, but will raise a warning in numpy 1.14 if numpy/numpy#9505 is merged